src/cui.cpp

```diff
@@ -387,8 +402,12 @@ void show_ncurses(Line *lines[], int nproc) {
   }
   attron(A_REVERSE);
   int totalrow = std::min(rows - 1, 3 + 1 + i);
-  mvprintw(totalrow, 0, "  TOTAL        %-*.*s %-*.*s    %11.3f %11.3f ",
-           proglen, proglen, "", devlen, devlen, "", sent_global, recv_global);
+  mvprintw(totalrow, 0, "  TOTAL        %-*.*s %-*.*s    %s %s ",
+           proglen, proglen, "", devlen, devlen, "", 
+	   prettyFloat(sent_global, NUMBER_OF_DECIMALS_SENT, COLUMN_WIDTH_SENT).c_str(), 
+	   prettyFloat(recv_global, NUMBER_OF_DECIMALS_RECEIVED, COLUMN_WIDTH_RECEIVED).c_str()
+	   );
+//	   recv_global);
```

Comment:
remove this comment

Reply:
i think i made the requested changes. If anything wrong let me know, bye.